tests/test_visualization.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pytest
+import os
+
+from continuum.datasets import MNIST
+from continuum.scenarios import Rotations
+from continuum.scenarios import Permutations
+from continuum.scenarios import ClassIncremental
+
+
+'''
+Test the visualization with three tasks
+'''
+def test_visualization_rotations():
+    clloader = Rotations(cl_dataset=MNIST("./pytest/Samples/Datasets", download=True, train=True),
```

Comment:
Can we do it with a dummy dataset? Downloading MNIST slows down the test pipeline.

Reply:
hum, yes

Reply:
I it is okay for you I will create a separate set of tests for the different datasets to evaluate if they works. We need to now, if there are bugs in download process or anything. we will not know it with dummy datasets only

Reply:
If you agree, I will do it in another PR after finishing this one :)

Reply:
Finalement, je pense que pour les tests de visualisation c'est pas mal de regarder avec des vrais données. Ca donne une meilleur apperçu de si le pipeline fonctionne normalement. Du coup j'ai garder les test tel qu'ils etaient mais je les ai marqué comme "slow" et j'ai modifié le Makefile pour que travis tourne pas ces tests la. Faudra juste les tourner une fois de temps en temps pour vérifier que ça marche toujours